docs/src/main/asciidoc/_admin-guide/Management_Clients.adoc

```diff
@@ -161,11 +161,34 @@ standard configuration files, e.g. `standalone-full.xml`,
 similar to the default `standalone.xml` file but includes additional
 subsystems not present in the default configuration. If you prefer to
 use one of these files as your server configuration, you can specify it
-with the _[line-through]*c*_ or _-server-config_ command line argument:
+with the *_-c_* or *_--server-config_* command line argument:
 
 * `bin/standalone.sh -c=standalone-full.xml`
 * `bin/standalone.sh --server-config=standalone-ha.xml`
 
+==== Aliases
+
+The configuration files can also be referenced by an alias instead of the file name:
+
+* `bin/standalone.sh -c ha`
+
+.Available aliases
+|===
+| Configuration file | Aliases
+
+| standalone-full.xml     | full
+
+| *-ha.xml                | ha
```

Comment:
@michpetrov What does '*' mean here and in the others?

Reply:
@michpetrov Why not just spell it out? Any symbol people will try and interpret what it means and may come up with novel interpretations, or will just wonder why they're needing to use mental energy.

Reply:
@bstansberry well, it's superfluous information and AFAIK it's not good UX to have each line start with the same word, but I'm not a docs person so if you don't mind it I'll keep it there